Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -975,11 +978,19 @@ def genS(fragment,howMuch):
 steps['SingleMuPt10FS_UP17']=merge([{'cfg':'SingleMuPt10_pythia8_cfi'},step1FastUpg2017Defaults])
 steps['SingleMuPt100FS_UP17']=merge([{'cfg':'SingleMuPt100_pythia8_cfi'},step1FastUpg2017Defaults])
 
+### FastSim: produce sample of minbias events for PU mixing
+steps['MinBiasFS_13_UP17_ForMixing']=merge([{'cfg':'MinBias_13TeV_pythia8_TuneCUETP8M1_cfi'},Kby(100,1000),step1FastPU17NewMixing])
+
 ### FastSim: template to produce signal and overlay with minbias events
 PUFS25={'--pileup':'AVE_35_BX_25ns',
         '--pileup_input':'das:/RelValMinBiasFS_13_ForMixing/%s/GEN-SIM-RECO'%(baseDataSetRelease[7],)}
 FS_UP15_PU25_OVERLAY = merge([PUFS25,Kby(100,500),steps['TTbarFS_13']] )
 
+### FastSim: template to produce signal and overlay with minbias events #PU50
+PUFSAVE50={'--pileup':'AVE_50_BX_25ns',
+        '--pileup_input':'das:/RelValMinBiasFS_13_UP17_ForMixing/%s/GEN-SIM-RECO'%(baseDataSetRelease[7],)}
+FS_UP17_PU50_OVERLAY = merge([PUFSAVE50,Kby(100,500),steps['TTbarFS_13_UP17']] )
+
```

Comment:
@zhenhu , the as the new MinBias sample is /RelValMinBiasFS_13_UP17_ForMixing/CMSSW_10_2_8-102X_mc2017_realistic_v5_FastSim-v1/GEN-SIM-RECO, I would add CMSSW_10_2_8-102X_mc2017_realistic_v5_FastSim-v1 to BaseDataSetRelease list.  I assume this won't break anything else?

Reply:
just to be clear, I think you only need to replace it in this line: https://github.com/cms-sw/cmssw/blob/3dbf7e41247832da54ec580a629fbb6f45460735/Configuration/PyReleaseValidation/python/relval_steps.py#L575

Reply:
indeed, that line is specific to the minbias sample.  Done.